src/router/graphql.ts

```diff
@@ -26,17 +26,26 @@ if (config.graphql) {
   // For a given DB name, either acquire the postgraphile middleware that already exists or create a
   // new one. No matter what, set the last access time to now so it is not terminated.
   const getPostgraphile = (db: string) => {
-    postgraphiles[db] = postgraphiles[db] ?? {
-      lastAccessMs: Date.now(),
-      graphile: postgraphile(
-        `postgresql://${config.db.user}:${config.db.password}@${config.db.host}/${db}${config.db.forceSSL ? '?sslmode=no-verify' : ''}`,
-        'public',
-        {
-          externalUrlBase: `/v1/graphql/${db}`,
-          graphiql: withGraphiql,
-          watchPg: true,
-        }),
-    };
+    try {
+      postgraphiles[db] = postgraphiles[db] ?? {
+        lastAccessMs: Date.now(),
+        graphile: postgraphile(
+          `postgresql://${config.db.user}:${config.db.password}@${config.db.host}/${db}${config.db.forceSSL ? '?sslmode=no-verify' : ''}`,
+          'public',
+          {
+            externalUrlBase: `/v1/graphql/${db}`,
+            graphiql: withGraphiql,
+            watchPg: true,
+          }),
+      };
+    } catch(e) {
+      // If postgraphile fails during construction, that error could crash the server. Instead
+      // we pass back an Express middleware that always fails
```

Comment:
I tried to connect to a local test database before I created it and crashed the server, so I added the fix to this PR to be quicker.